politeiawww/backend.go

```diff
@@ -1041,3 +1075,13 @@ func NewBackend(cfg *config) (*backend, error) {
 
 	return b, nil
 }
+
+// getProposalName returns the proposal name based on the index markdown file.
+func getProposalName(files []www.File) (string, error) {
+	for _, file := range files {
+		if file.Name == indexFile {
+			return util.ProposalName(file.Payload)
+		}
+	}
+	return "", nil
+}
```

Comment:
We need to limit what we allow in the title.  How does this sound?
[a-z][A-Z][0-9][@#+-:;,.]

Reply:
covered